components/time/src/zone/mod.rs

```diff
@@ -95,16 +72,16 @@ pub use offset::VariantOffsetsCalculator;
 pub use offset::VariantOffsetsCalculatorBorrowed;
 
 #[doc(no_inline)]
-pub use iana::IanaParser;
+pub use iana::{IanaParser, IanaParserBorrowed};
```

Comment:
Thought: I don't know why we are pulling out `Borrowed` but not the iterators. I feel like `IanaParser` has 9 call sites for every 3 of `IanaParserBorrowed` and 2 of `IanaParserIterator`. If the line is "the name is likely to be used", then we shouldn't pull out Borrowed.

@robertbastian points out that in most places we've been promoting `Borrowed`.

@Manishearth

Reply:
I think Borrowed is the type people are going to pass around a lot since people will be doing compiled data. I think having Borrowed be toplevel everywhere is a norm we have been following that makes some sense to me.

Iterators are on the other hand a type that will be referenced only when making wrappers.